plotsearch/migrations/0032_init_area_search.py

```diff
@@ -0,0 +1,22 @@
+from django.db import migrations
+from forms.models import FieldType
+from plotsearch.utils import initialize_area_search_form
+
+
+def init_area_search_form(apps, schema_editor):
+    if 8 <= FieldType.objects.all().count():
```

Comment:
What is this magic number 8? It would be nice to have it in a variable that has explanatory name :)

Reply:
Checking that field types exist, but changed the checking and added actually creating field types when needed.